sdc-plugin/sdc.cc

```diff
@@ -192,18 +192,59 @@ struct GetClocksCmd : public Pass {
 	log("\n");
     }
 
+    std::vector<std::string> extract_list(const std::string& args) {
```

Comment:
Is there a way to leverage the TCL interpreter's list processing here?  This doesn't seem bad, but I wonder if there will be some TCL <-> this implemenation mismatch?

Reply:
We probably could, but I am filtering the content of the list later on before packing the final TCL list, so the use of a Tcl_Object would be a bit artificial in my opinion.

Reply:
I guess my point is that if the `list` that this function is extracting is a Tcl list, I suspect believe your implementation does not respects the Tcl parsing rules properly.

Reply:
Could you give an example? `get_clocks -of [list a b c]` and `get_clocks -of [get_nets ...]` works in the current implementation.

Reply:
`get_clocks -of [concat [get_nets c] [get_nets {a b}]]` ?

Reply:
> `get_clocks -of [concat [get_nets c] [get_nets {a b}]] `

[this](https://github.com/SymbiFlow/yosys-symbiflow-plugins/pull/52/files#diff-66206c3e9c4388d75399de4a47827146bf328f9738dd60b32606c948c9175d86R33) works